### PR TITLE
Add Bitdefender.app v2.3

### PR DESCRIPTION
**Pull request by dcondrey**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

**Comment by dcondrey** [comment]
The dmg file contains a secondary uninstaller pkg which is not installed with the installer package, but seems to do a more thorough uninstall than what I wrote in the stanza.  I'm not sure how to reference that secondary uninstall pkg though.

**Comment by miccal** [comment]
> I'm not sure how to reference that secondary uninstall pkg though.

You can't at this stage - see discussion [here](https://github.com/caskroom/homebrew-cask/issues/24377).

**Comment by dcondrey** [comment]
I thought perhaps I could copy over the uninstall package as a post_install step but not sure.

**Comment by dcondrey** [comment]
@fanquake Why the on hold tag?

**Comment by adidalal** [comment]
Does the app fully uninstall? If not, we can't add it - that's why the tag, I believe.

**Comment by dcondrey** [comment]
The app does uninstall successfully as is.

File: Casks/bitdefender.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'bitdefender' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://download.bitdefender.com/mac/antivirus/en/bitdefender_antivirus_for_mac.dmg'
+  name 'Bitdefender Virus Scanner'
+  homepage 'http://www.bitdefender.com/solutions/virus-scanner-for-mac.html'
```
Comment (miccal) [inline Casks/bitdefender.rb]: Can you confirm this is the correct homepage? I get a 404 error, and going to `http://www.bitdefender.com.au/solutions/antivirus-for-mac.html` I need to enter an email address to download the trial, which is not the one downloaded via the `url` you provided.

Comment (Numbermaniac) [inline Casks/bitdefender.rb]: The given homepage works just fine, testing with an American VPN, but as you said the .au version fails for me too. Maybe we should just make this `http://www.bitdefender.com/`?

Comment (miccal) [inline Casks/bitdefender.rb]: But even from `http://www.bitdefender.com/` I cannot find a link to the file in the original `url`?

Comment (Numbermaniac) [inline Casks/bitdefender.rb]: Is this maybe a link to the paid version, then?

Comment (miccal) [inline Casks/bitdefender.rb]: @dcondrey any follow-up on this?

Comment (dcondrey) [inline Casks/bitdefender.rb]: All links on the website direct to the App Store but I found this direct link, crawling around for it with a little help from Google.

**Comment by dcondrey** [comment]
All links on the website direct to the App Store but I found this direct link, crawling around for it with a little help from Google.

**Comment by miccal** [comment]
Thanks @dcondrey - unfortunately, unless there is an easily verifiable link via the `homepage`, I do not think we can accept this.

**Comment by reitermarkus** [comment]
I found it here: http://www.bitdefender.com/Downloads/

**Comment by miccal** [comment]
@reitermarkus those links do not download the `.pkg` referenced in the original PR - the `.dmg contains a `.pkg` called `Bitdefender_aa9d5ce5-7b01-4743-9eec-5480ce596363.pkg`.

**Comment by reitermarkus** [comment]
You're right, seems like the trial really is only a trial.

- `http://download.bitdefender.com/mac/av/en/bitdefender_antivirus_for_mac.dmg` (15.1 MB)
- `http://download.bitdefender.com/mac/antivirus/en/bitdefender_antivirus_for_mac.dmg` (171.4 MB)

**Comment by miccal** [comment]
Precisely.

**Comment by dcondrey** [comment]
Downloaded this new one and it installs an obnoxious app called BitDefender Central which just seems to be an intermediary forcing me to create a user account.

**Comment by reitermarkus** [comment]
The 171.4 MB one seems to be from 2012.

**Comment by miccal** [comment]
Ah, vintage!

**Comment by dcondrey** [comment]
This Bitdefender software kinda seems to be it's own virus.. it takes up way too much space and resources. :/

**Comment by miccal** [comment]
Thank you for your efforts though @dcondrey, they are appreciated.

**Comment by dcondrey** [comment]
np

**Pull request closed without merging**